docs/modules/ROOT/pages/features.adoc

```diff
@@ -0,0 +1,338 @@
+= Features
+:url-asciidoctor-sc-image: https://cdn.jsdelivr.net/gh/asciidoctor/asciidoctor/screenshot.png
+:source-language: asciidoc
+
+The main feature is viewing AsciiDoc as HTML5 directly in the browser.
+Other features cover where those AsciiDoc files are located, and how they are rendered.
```

Comment:
I don't know what you mean by "where those AsciiDoc files are located"?

I would say something like: "Other features include automatic reload, built-in and custom themes, AsciiDoc attributes as query parameters, diagrams using Kroki and more."